src/core/Directus/Services/AssetService.php

```diff
@@ -114,18 +98,11 @@ public function getThumbnail($params)
         $this->thumbnailParams=$params;
 
         $this->validateThumbnailParams($params);
-
+      
```

Comment:
You should enable the "Remove trailing whitespace" setting in your editor to prevent empty changes like this